android/src/main/java/io/parity/signer/ui/rootnavigation/RootNavigationGraph.kt

```diff
@@ -81,23 +81,16 @@ fun NavGraphBuilder.firstTimeOnlyOnboarding(
 ) {
 	navigation(
 		route = routePath,
-		startDestination = FirstTimeOnboarding.onboardingExplanationRoute,
+		startDestination = FirstTimeOnboarding.termsConsentRoute,
 	) {
-		composable(route = FirstTimeOnboarding.onboardingExplanationRoute) {
-			OnboardingExplanationScreenFull(navigateNext = {
-				navController.navigate(
-					FirstTimeOnboarding.termsConsentRoute
-				)
-			})
-		}
 		composable(route = FirstTimeOnboarding.termsConsentRoute) {
 			TermsConsentScreenFull(navigateNextScreen = onNextStepsNavigate)
 		}
 	}
 }
 
 private object FirstTimeOnboarding {
-	const val onboardingExplanationRoute = "navigation_onboarding_explanation"
+//	const val onboardingExplanationRoute = "navigation_onboarding_explanation"
```

Comment:
Shouldn't this be removed, given we have git history to bring it back?

Reply:
will remove on subsequent redesign pr